themes.gitconfig

```diff
@@ -167,6 +167,9 @@
     syntax-theme = Vibrant Sunburst
     whitespace-error-style = "#280050" reverse
     zero-style = syntax
+    blame-format = "{author:<18} ({commit:>7}) ┊{timestamp:^16}┊ "
+    ;blame-timestamp-format = "%Y-%m-%d %H:%M:%S %z"
```

Comment:
I asked that weeks ago but you never saw it :/ Don't worry, I'll remove the comment and merge this now. Thanks!

Reply:
Thank you ! :smile:

I have commented that out because I wasn't sure if time formats should be part of a color theme. Do that it wouldn't be activated by default, but users could easily turn it on.

Anyways, if you are are fine with it being uncommented then so am I. ;)